src/core/components/Root/index.js

```diff
@@ -12,18 +13,30 @@ import type { ReactRouterHistoryType } from 'core/types/router';
 type Props = {|
   children: React.Node,
   cookies?: typeof Cookies,
+  _config: typeof config,
   history: ReactRouterHistoryType,
   i18n: I18nType,
   store: ReduxStore,
 |};
 
-const Root = ({ children, history, i18n, store, cookies = null }: Props) => (
+const Root = ({
+  _config,
+  children,
+  history,
+  i18n,
+  store,
+  cookies = null,
+}: Props) => (
   <I18nProvider i18n={i18n}>
     <Provider store={store} key="provider">
       <ConnectedRouter history={history}>
         <CookiesProvider cookies={cookies}>
           {/* $FLOW_FIXME: https://github.com/facebook/react/issues/12553 */}
-          <React.StrictMode>{children}</React.StrictMode>
+          {_config.get('enableStrictMode') ? (
```

Comment:
**Note:** Check out where to set the value `enableStrictMode` as per other examples in one of these files:

![image](https://user-images.githubusercontent.com/13009507/48924784-00232180-ee8a-11e8-87ab-cbac792c8d2e.png)

Reply:
https://travis-ci.org/mozilla/addons-frontend/jobs/458615128#L5310 says that `config` is undefined